### PR TITLE
#TAS-63 - Bump FOSElastica

### PR DESCRIPTION
**Pull request by TheSylwio**
Closes #29

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -5,9 +5,9 @@
     "description": "CrossSelling plugin for Sylius applications.",
     "license": "MIT",
     "require": {
-        "php": "^7.3 || ^7.4 || ^8.0",
+        "php": "^8.0",
         "bitbag/coding-standard": "^1.0",
-        "friendsofsymfony/elastica-bundle": "^5.2",
+        "friendsofsymfony/elastica-bundle": "^6.0",
```
Comment (senghe) [inline composer.json:10]: Why not ^6.1? I think better would be to update dependencies into newest version if possible

File: tests/Behat/Service/ElasticsearchCommands.php [hunk 2]
```diff
@@ -70,19 +63,21 @@ public function populateAllIndexes(): void
         ];
 
         foreach ($indexes as $index) {
-            $event = new IndexPopulateEvent($index, true, $options);
-            $this->dispatcher->dispatch($event, IndexPopulateEvent::PRE_INDEX_POPULATE);
```
Comment (senghe) [inline tests/Behat/Service/ElasticsearchCommands.php:74]: Why we change pre populating into post populating?

File: tests/Unit/PropertyBuilder/RelatedProductsPropertyBuilderTest.php [hunk 3]
```diff
@@ -13,16 +13,15 @@
...
 use PHPUnit\Framework\TestCase;
 use Sylius\Component\Core\Model\OrderInterface;
 use Sylius\Component\Core\Model\OrderItemInterface;
 use Sylius\Component\Core\Model\ProductInterface;
 
 final class RelatedProductsPropertyBuilderTest extends TestCase
 {
-    /** @var RelatedProductsPropertyBuilder */
-    private $sut;
+    private RelatedProductsPropertyBuilder $sut;
```
Comment (senghe) [inline tests/Unit/PropertyBuilder/RelatedProductsPropertyBuilderTest.php:24]: Maybe better would be to rename it, as the `$sut` doesn't tell me anything about it? :)

**Review by senghe — CHANGES_REQUESTED** [review]

File: tests/Behat/Service/ElasticsearchCommands.php (hunk 2 above)
Comment (senghe) [reply tests/Behat/Service/ElasticsearchCommands.php:74]: Bump?

Comment (TheSylwio) [reply tests/Behat/Service/ElasticsearchCommands.php:74]: Changing `PRE_INDEX_POPULATE` into `POST_INDEX_POPULATE` was my oversight. I changed it here: https://github.com/BitBagCommerce/SyliusCrossSellingPlugin/pull/32/commits/a1a2a2902dee5184f65b59f9a1db360dc974b08c

Comment (senghe) [reply tests/Behat/Service/ElasticsearchCommands.php:74]: Ok, so in the end, we should use POST_INDEX_POPULATE? As it's currently implemented

**Pull request closed without merging**